[2024-11-18T20:20:17.900+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-18T20:20:17.966+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_api_sales_data.insert_sales_data scheduled__2024-11-18T20:10:00+00:00 [queued]>
[2024-11-18T20:20:18.008+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_api_sales_data.insert_sales_data scheduled__2024-11-18T20:10:00+00:00 [queued]>
[2024-11-18T20:20:18.011+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-11-18T20:20:18.085+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): insert_sales_data> on 2024-11-18 20:10:00+00:00
[2024-11-18T20:20:18.109+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3642) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-18T20:20:18.118+0000] {standard_task_runner.py:72} INFO - Started process 3644 to run task
[2024-11-18T20:20:18.123+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'raw_api_sales_data', 'insert_sales_data', 'scheduled__2024-11-18T20:10:00+00:00', '--job-id', '439', '--raw', '--subdir', 'DAGS_FOLDER/raw/__pycache__/fetch_data.py', '--cfg-path', '/tmp/tmp5iz57l3s']
[2024-11-18T20:20:18.128+0000] {standard_task_runner.py:105} INFO - Job 439: Subtask insert_sales_data
[2024-11-18T20:20:18.366+0000] {task_command.py:467} INFO - Running <TaskInstance: raw_api_sales_data.insert_sales_data scheduled__2024-11-18T20:10:00+00:00 [running]> on host c06adc74012b
[2024-11-18T20:20:19.413+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_api_sales_data' AIRFLOW_CTX_TASK_ID='insert_sales_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T20:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-18T20:10:00+00:00'
[2024-11-18T20:20:19.416+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-18T20:20:19.418+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-18T20:20:19.420+0000] {logging_mixin.py:190} INFO - Current task name:insert_sales_data state:running start_date:2024-11-18 20:20:17.968113+00:00
[2024-11-18T20:20:19.422+0000] {logging_mixin.py:190} INFO - Dag name:raw_api_sales_data and current dag run status:running
[2024-11-18T20:20:19.423+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-18T20:20:19.500+0000] {base.py:84} INFO - Retrieving connection 'postgres_default'
[2024-11-18T20:20:19.636+0000] {fetch_data.py:73} INFO - Sales data successfully inserted into PostgreSQL.
[2024-11-18T20:20:19.641+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-18T20:20:19.700+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-18T20:20:19.701+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=raw_api_sales_data, task_id=insert_sales_data, run_id=scheduled__2024-11-18T20:10:00+00:00, execution_date=20241118T201000, start_date=20241118T202017, end_date=20241118T202019
[2024-11-18T20:20:19.735+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-18T20:20:19.737+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-18T20:20:19.739+0000] {logging_mixin.py:190} INFO - Dag name:raw_api_sales_data queued_at:2024-11-18 20:20:00.061555+00:00
[2024-11-18T20:20:19.741+0000] {logging_mixin.py:190} INFO - Task hostname:c06adc74012b operator:PythonOperator
[2024-11-18T20:20:19.795+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-18T20:20:19.928+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-18T20:20:19.957+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
